SCons/Defaults.xml

```diff
@@ -509,22 +528,33 @@ env = Environment(LINKCOM="my_linker $_LIBDIRFLAGS $_LIBFLAGS -o $TARGET $SOURCE
 A list of one or more libraries
 that will be linked with
 any executable programs
-created by this environment.
+created by this environment,
+or if used in an override,
```

Comment:
most users won't know what an override is.
Maybe say you can add this explicitly to Library() or Program() to affect only that target?

Reply:
The term is "defined" in https://scons.org/doc/4.1.0/HTML/scons-man.html#builder_methods, if you scroll down to the paragraph which begins wtih

> Keyword arguments that are not

And yes, can add to some common place.

Reply:
The Construction Variables section got a mention of overrides, removed from here.